src/helm/benchmark/augmentations/translate_perturbation.py

```diff
@@ -0,0 +1,29 @@
+from dataclasses import dataclass
+from random import Random
+
+from helm.proxy.clients.google_translate_client import GoogleTranslateClient
+from .perturbation import TextPerturbation
+from .perturbation_description import PerturbationDescription
+
+
+class TranslatePerturbation(TextPerturbation):
+    """
+    Translates to different languages.
+    """
+
+    @dataclass(frozen=True)
+    class Description(PerturbationDescription):
+        language_code: str = "zh-CN"
```

Comment:
Remove the default value for `language_code` (not sure why "zh-CN" should be the default).